docs/spec/literal.rst

```diff
@@ -98,16 +98,16 @@ expressions, and nothing else.
 Legal parameters for ``Literal`` at type check time
 """""""""""""""""""""""""""""""""""""""""""""""""""
 
-``Literal`` may be parameterized with literal ints, byte and unicode strings,
+``Literal`` may be parameterized with literal ints, strings, `bytes` objects,
```

Comment:
For consistency, maybe use the class names for `int`, `str`, `bytes`, and `bool`. With the proposed change, only `bytes` uses the actual class name.

Also, I think this requires double backticks.